elastic/elastic-agent
# Endpoint should be allowed to run on Windows Server 2012/2012R2

## What does this PR do?

In 8.16 we are adding back Elastic Defend support for Windows Server 2012 and 2012 R2 as they are still on extended support from Microsoft.

## Why is it important?

There are customers who can't upgrade yet to newer Windows Server operating systems so are paying for extended Windows support. In such case it's possible to run an older Endpoint with newer stack, but it also requires older version of Agent and all Beats, therefore we're changing our strategy.

## Checklist




## Disruptive User Impact



## How to test this PR locally

Package Agent with latest snapshoot build of Endpoint. Install on Windows Server 2012 or 2012 R2. Add Elastic Defend integration and verify that it works.

## Related issues


-

## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...
